CONTRIBUTING.md

```diff
@@ -107,6 +112,10 @@ More information on the change process and change types can be found on the [bea
 If you are addressing multiple issues which are unrelated, consider either doing multiple pull requests, or generate separate change files to ensure accurate generation of changelogs and versioning of packages.
 :::
 
+:::note
+If you are finding that your changes are either breaking changes or require multiple pull requests, open an issue to discuss this.
```

Comment:
Suggestion to replace "open an issue to discuss this" with "open a [discussion](https://github.com/microsoft/fast/discussions)".